.github/workflows/javadoc.yml

```diff
@@ -3,21 +3,28 @@ name: JavaDoc
 on:
   push:
     branches:
-      - 2.2.X
+      - 4.0.X
```

Comment:
I think this means it will only run on the 4.0.x branch and would need manual dispatch for anything else? 3.0.x, 4.1.x, etc?

Reply:
It was originally 2.2.X.  I don't know who put it there, and it didn't make sense to me either.  I would just leave it as manual only.

Reply:
Made it `main` and it'll otherwise be manual